3rd-party-licenses.md

```diff
@@ -6,6 +6,9 @@ The Pimcore project is licensed under the terms of the [GPLv3](LICENSE.md).
 However, Pimcore includes several third-party Open-Source libraries, 
 which are licensed under their own respective Open-Source licenses.
 
+The specific verions of a library are documented in
+[PIMCORE_MODIFICATIONS.md](bundles/AdminBundle/Resources/public/js/lib/PIMCORE_MODIFICATIONS.md).
```

Comment:
```suggestion
[PIMCORE_MODIFICATIONS.md](https://github.com/pimcore/pimcore/tree/11.x/bundles/AdminBundle/Resources/public/js/lib/PIMCORE_MODIFICATIONS.md).
```

Reply:
@dvesh3 Is this actually a good idea? We would need to update that for every future branching change.

The way I wrote it should work for future changes where the path has not changed and forked repos, etc. WDYT?

Reply:
yes, you are right. I misunderstood it with docs so should be fine like this.